.distignore

```diff
@@ -22,6 +24,7 @@ CREDITS.md
 LICENSE.md
 package-lock.json
 package.json
+phpcs.xml
 README.md
 webpack.config.js
 wpacceptance.json
```

Comment:
I think we'll want to leave `wpacceptance.json` for now, but looks like somehow it and the associated WPA bits and bobs are still in `trunk` (e.g., https://github.com/10up/insert-special-characters/blob/trunk/.github/workflows/test.yml && https://github.com/10up/insert-special-characters/tree/trunk/tests/wpacceptance).  Will see if the merge from `develop` to `trunk` properly removes them, but if not we should open another PR to remove those from `trunk` so they don't accidentally sneak into a future release to WP.org.